R/Shiny.R

```diff
@@ -110,128 +110,59 @@ launchDiagnosticsExplorer <- function(dataFolder = "data",
     connectionDetails = connectionDetails,
     resultsDatabaseSchema = resultsDatabaseSchema,
     vocabularyDatabaseSchemas = vocabularyDatabaseSchemas,
-    dataFolder = dataFolder,
-    dataFile = dataFile,
     aboutText = aboutText
   )
   .GlobalEnv$shinySettings <- shinySettings
   on.exit(rm("shinySettings", envir = .GlobalEnv))
   shiny::runApp(appDir = appDir)
 }
 
-#' Premerge Shiny diagnostics files
+#' Merge Shiny diagnostics files into sqlite database
 #'
 #' @description
-#' This function combines diagnostics results from one or more databases into a single file. The result is a
-#' single file that can be used as input for the Diagnostics Explorer Shiny app.
+#' This function combines diagnostics results from one or more databases into a single file. The result is an
+#' sqlite database that can be used as input for the Diagnostics Explorer Shiny app.
 #'
 #' It also checks whether the results conform to the results data model specifications.
 #'
 #' @param dataFolder       folder where the exported zip files for the diagnostics are stored. Use
 #'                         the \code{\link{runCohortDiagnostics}} function to generate these zip files.
 #'                         Zip files containing results from multiple databases may be placed in the same
 #'                         folder.
-#' @param tempFolder       A folder on the local file system where the zip files are extracted to. Will be cleaned
```

Comment:
maybe we couldc, if tempFolder parameter is being used

- put a stop error (not warning)
- but before stopping - check if there is preMergedFile in the tempFolder
- state that premerged file support was stopped in version x and after, and replace with MergedCohortDiagnosticsData.sqlite.
- show user how to create MergedCohortDiagnosticsData.sqlite